src/facade/reply_builder.h

```diff
@@ -12,6 +12,12 @@
 #include "io/io.h"
 
 namespace facade {
+// some constants, used in reply_builder.cc; can be used elsewhere.
+namespace constants {
+constexpr char kCRLF[] = "\r\n";
+constexpr char kErrPref[] = "-ERR ";
+constexpr char kSimplePref[] = "+";
```

Comment:
While here, I'd convert these to `string_view` (e.g: `constexpr string_view kCRLF = "\r\n"sv`).